gh2/csv.py

```diff
@@ -104,29 +105,36 @@ def is_pull_request(issue):
 
 
 def normalize_sequential_dates(issue_list):
-    for start, item in enumerate(issue_list):
-        if isinstance(item, datetime.datetime):
+    start, finish = ('created_at', 'closed_at')
+    date_fields = []
+    for field in issue_list:
+        if field == start:
+            date_fields.append(field)
+            continue
+        elif not date_fields:
```

Comment:
I don't believe that was the assumption prior to this (that they be contiguous). Since we're now enforcing that, can we add a comment here to indicate that?